### PR TITLE
add rails example

### PR DESCRIPTION
**Pull request by shunsuke-h-opn**
#### 1. Objective
I added rails for Omise.

#### 2. Description of change
These are what I added.
Rails(Credit Card Charge, PayPay, Google Payment)

#### 3. Impact of the change
None.

#### 4. Priority of change
Normal.

### CODE REVIEW

File: ruby_rails/README.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+
+# Requirements
+To follow along, you will need:
+
+An Omise account (and your $OMISE_PUBLIC_KEY and $OMISE_SECRET_KEY)
+
+This example project use this Ruby and Rails version
+
+- Ruby 3.0.0
```
Comment (danfowler) [inline ruby_rails/README.md]: 3.0 end of life is in four months, so maybe we should start with a higher version?

Comment (shunsuke-h-opn) [reply ruby_rails/README.md]: @danfowler 
Oh, I think you are right.
I will apply higher ruby.

Comment (shunsuke-h-opn) [reply ruby_rails/README.md]: https://github.com/omise/examples/pull/30/commits/1e254aa2596ae8dafdc824a080055002bf2b4214

Comment (shunsuke-h-opn) [reply ruby_rails/README.md]: https://github.com/omise/examples/pull/30/commits/26a9b60540d502ff93617c04bc8355152c7567db

**Review by danfowler — APPROVED** [review]

**Review by aashishgurung — APPROVED** [review]

**Pull request merged**